api/models/workshop.py

```diff
@@ -23,6 +25,10 @@ class Workshop(CapacityMixin, Base):
         WorkshopDifficulty,
         verbose_name=_("Difficulty"),
     )
+    location = models.ForeignKey(
```

Comment:
@PetrDlouhy, yes. Also, `Workshop` can already be reused over the years. This change breaks that. What do you mean a LOT of changes? Do you mean having this relation as many2many over a year? Well, I guess that's just insane.

Changing lector logically changes the workshop content as well.